engines/glk/glulx/detection_tables.h

```diff
@@ -1176,7 +1176,7 @@ const PlainGameDescriptor GLULXE_GAME_LIST[] = {
 	{ "pc21_grooverland",   "Grooverland" },                                                            //    1st Place
 	{ "pc21_foreignsoil",   "Foreign Soil" },                                                           //    2nd Place
 	{ "pc21_blackknife",    "Black Knife Dungeon" },                                                    //    3rd Place
-	{ "pc21_timemachi21",   "The Time Machine" },                                                       //    6th Place
+	{ "pc21_timemachi21",   "The Time Machine (by Bill Maya)" },                                        //    6th Place
```

Comment:
Why are you including the author's name in the title? The author himself calls it "The Time Machine"
https://billmaya.itch.io/the-time-machine

Reply:
There are cases where adventures  with different author, release date, hash (etc etc...) share only the title.

In this case there are two "The Time Machine" games (one Adrift and one Glulx), in other the same "duplicated" title occurs in the same subengine so I cannot keep them in tha same dir.

As done in previuos PRs, I've choosen to follow this path to differentiate them. What do you suggest to achieve a better approach?

Reply:
Ah, I see, thanks for the explanation! :)